type-definitions/immutable.js.flow

```diff
@@ -561,9 +561,78 @@ declare module 'immutable' {
     flatten(shallow?: boolean): /*this*/Map<any,any>;
   }
 
-  // OrderedMaps have nothing that Maps do not have. We do not need to override constructor & other statics
-  declare class OrderedMap<K,V> extends Map<K,V> {
+  declare class OrderedMap<K,V> extends KeyedCollection<K,V> {
```

Comment:
Why did you decide to extend `KeyedCollection`? `OrderedMap` [extends `Map`](https://github.com/facebook/immutable-js/blob/master/src/OrderedMap.js#L17).